rustfmt.toml

```diff
@@ -1,9 +1,9 @@
-edition = "2021"
+edition = "2024"
 max_width = 120
 fn_params_layout = "Tall"
 reorder_imports = true
-# group_imports = "StdExternalCrate"
-# imports_granularity = "Crate"
+group_imports = "StdExternalCrate"
+imports_granularity = "Module"
 
 # nightly features:
 #ignore = ["db/entity/src/codegen", "ethereum/bindings/src/codegen"]
```

Comment:
The codegen should be ignored though, or the check in CI will fail with changes each time (or it should).

Reply:
The ignoring is done in the treefmt-nix configuration in `flake.nix`.